keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/cluster/TitusClusterHandler.kt

```diff
@@ -172,7 +172,8 @@ class TitusClusterHandler(
     )
   }
 
-  fun generateContainer(container: ContainerWithDigest, account: String): Container {
+  // todo eb: this should generate a reference...
```

Comment:
This is a todo for another PR: essentially, fix cluster export to work with artifact references etc.